courses/dl2/cifar10-darknet.ipynb

```diff
@@ -79,7 +79,10 @@
     "        self.conv1=conv_layer(ni, ni//2, ks=1)\n",
     "        self.conv2=conv_layer(ni//2, ni, ks=3)\n",
     "        \n",
-    "    def forward(self, x): return x.add_(self.conv2(self.conv1(x)))"
+    "    def forward(self, x): \n",
+    "        # When used with DataParallel on a single GPU, we can't use the inplace version of x.add_, which leads to error:\n",
```

Comment:
Not sure this is linked to one GPU. In general in_place operations and variables require gradients don't always mix well and pytorch raises an error when it isn't fully sure it has all the proper graph for gradients computation. I'd change that comment.

Reply:
The comment applies to a specific error: "one of the variables needed for gradient computation has been modified by an inplace operation". Perhaps listing the exact error for a reason why `add_` wasn't used would do the right thing? I can't test whether `add_` does work with DataParallel, since i only have one GPU, but it must have worked for Jeremy. That's why I phrased it this way in first place.

Reply:
Most likely the pytorch version is different as I don't see why the number of GPUs would affect this error. I'd leave the comment to 'using add_ leads to error (...) so we use add instead.'

Reply:
Same old 0.3. I thought that was the requirement for fastai v0. Unless Jeremy used a higher version when running this notebook.

I will change the comment.

Reply:
Done. And I added a simple statement that it happened when run on single GPU, without saying that this was the culprit. Hope this is reasonable.